### ISSUE TITLE
Error in Exporting Tier 2 Bacpac file

### ISSUE THREAD

**Issue by WajahatHussain5** [d365collaborative/d365fo.tools#779]
I encountered during the execution of a Tier 2 database export operation using the provided PowerShell command. The error message received is as follows:
 **Something went wrong while cloning the Azure DB database. | Exception calling "ExecuteNonQuery" with "0" argument(s): "CREATE DATABASE permission denied in database 'master'."** 
While with the same credentials and same VM. I am able to connect SSMS (SQL Server Management Studio).
Used below command for tier 2 with my credentials. 
"New-D365Bacpac -ExportModeTier2 -DatabaseServer dbserver1.database.windows.net -DatabaseName AxDB -SqlUser User123 -SqlPwd "Password123" -NewDatabaseName Testing1 -BacpacFile C:\Temp\Bacpac\Testing1.bacpac"

Before tier 2, I had successfully exported .bak file from tier 1 command (Local AxDB). I kindly request assistance in resolving this issue.

**Comment by FH-Inway** [comment]
As part of the export, [New-D365Bacpac](https://github.com/d365collaborative/d365fo.tools/blob/e197350b29923c053b251521391801766ff29207/d365fo.tools/functions/new-d365bacpac.ps1#L311-L330) runs a SQL script and creates a backup copy of the database. Both require write access to the database. The error message seems to indicate that only read access is available.

If [Enable just-in-time database access](https://learn.microsoft.com/en-us/dynamics365/fin-ops-core/dev-itpro/database/database-just-in-time-jit-access) is being used, try requesting access with write permissions.

The other option could be to use the [-ExportOnly](https://github.com/d365collaborative/d365fo.tools/blob/e197350b29923c053b251521391801766ff29207/d365fo.tools/functions/new-d365bacpac.ps1#L48-L49) switch so only read access is required. Note that in that case, the preparatory [clear-azurebacpacdatabase.sql](https://github.com/d365collaborative/d365fo.tools/blob/e197350b29923c053b251521391801766ff29207/d365fo.tools/internal/sql/clear-azurebacpacdatabase.sql) script will not be executed.

**Comment by WajahatHussain5** [comment]
Hi @***@***.*** ***@***.***>

I appreciate your help,  and figured it out that was due to read
permission, I had granted with. Therefore when I use export only command it
worked. But I am facing another issue now which is ofcourse not related to
your projects. Taking Teir2 environment is too slow that Just in time
access got terminated. Because bacpac file creating a lot of time,
yesterday night, connection terminated when 30 gb bacpac file has created .
Though I was monitoring its size was increasing and total bacpac supposed
to be 75 - 80 gb. Is there any resume mechanism once connection terminate
for the same bacpac file?

I really  appreciate  your response.

Thanks & Regards
Syed Wajahat
Microsoft Dynamic 365 Solution Engineer
